examples/wine-flavors.get.json

```diff
@@ -0,0 +1,95 @@
+{
+    "wine_flavors": [
```

Comment:
I've searched a bit, "flavor" might not be the correct wording, I suggest to change wording to "**taste**" or "characteristics"

Ref: https://winefolly.com/deep-dive/wine-characteristics/

Reply:
The term "wine characteristics" seems appropriate.